src/main/java/org/folio/event/service/UserEventProducer.java

```diff
@@ -40,39 +41,47 @@ public UserEventProducer(KafkaConfig kafkaConfig) {
   }
 
   public Future<Boolean> sendUserEvent(User user,
+                                       boolean isPersonalDataChanged,
                                        UserEvent.Action eventAction,
                                        Map<String, String> okapiHeaders) {
     String tenantId = okapiHeaders.get("x-okapi-tenant");
-    UserEvent event = getUserEvent(user, tenantId, eventAction);
+    UserEvent event = getUserEvent(user, tenantId, isPersonalDataChanged, eventAction);
     String eventPayload = Json.encode(event);
 
-    switch (eventAction) {
-      case CREATE:
-        logger.info("Starting to send user created event with id: {} for User to Kafka for userId: {}", event.getId(), user.getId());
-        return sendToKafka(UserEventType.USER_CREATED, eventPayload, okapiHeaders, user.getId());
-      case DELETE:
-        logger.info("Starting to send user deleted event with id: {} for User to Kafka for userId: {}", event.getId(), user.getId());
-        return sendToKafka(UserEventType.USER_DELETED, eventPayload, okapiHeaders, user.getId());
-      case EDIT:
-        logger.info("Starting to send user edit event with id: {} for User to Kafka for userId: {}", event.getId(), user.getId());
-        return sendToKafka(UserEventType.USER_UPDATED, eventPayload, okapiHeaders, user.getId());
-      default:
-        throw new IllegalStateException("Unexpected value: " + eventAction);
-    }
+      return switch (eventAction) {
+          case CREATE -> {
+              logger.info("Starting to send user created event with id: {} for User to Kafka for userId: {}", event.getId(), user.getId());
+              yield sendToKafka(UserEventType.USER_CREATED, eventPayload, okapiHeaders, user.getId());
+          }
+          case DELETE -> {
+              logger.info("Starting to send user deleted event with id: {} for User to Kafka for userId: {}", event.getId(), user.getId());
+              yield sendToKafka(UserEventType.USER_DELETED, eventPayload, okapiHeaders, user.getId());
+          }
+          case EDIT -> {
+              logger.info("Starting to send user edit event with id: {} for User to Kafka for userId: {}", event.getId(), user.getId());
+              yield sendToKafka(UserEventType.USER_UPDATED, eventPayload, okapiHeaders, user.getId());
+          }
+          default -> throw new IllegalStateException("Unexpected value: " + eventAction);
+      };
```

Comment:
Formatting issue, 2 spaces needed instead of 4

Reply:
Formatted